docs/tutorial/part-one/index.md

```diff
@@ -403,6 +405,28 @@ Once this finishes running, you should see in your terminal something like:
 Open the web address listed on the bottom line (`lowly-pain.surge.sh` in this
 case) and you'll see your newly published site! Great work!
 
+## Alternative: Deploying to Gatsby Cloud
+
+[Gatsby Cloud](https://gatsbyjs.com) is a platform built specifically for Gatsby sites, with features like real-time previews, fast builds, and integrations with dozens of other tools.
+
+To deploy your site to Gatsby Cloud, create an account on [Github](https://github.com) if you don’t have one. Github allows you to host and collaborate on code projects using Git for version control.
+
+Create a new repository on Github. Since you’re importing your existing project, you’ll want a completely empty one, so don’t initialize it with `README` or `.gitignore` files.
+
+You can tell Git where the remote (i.e. not on your computer) repository is like this:
+
+```
+git remote add origin [GITHUB_REPOSITORY_URL]
+```
+
+When you created a new Gatsby project with a starter, it automatically made an initial `git commit`, or a set of changes. Now, you can push your changes to the new remote location:
+
+```
+git push -u origin master
```

Comment:
Are we certain that the user doesn't need to directly set/state upstream on the first push?

Reply:
I'm not sure what you mean here. Since this is the first push I think the user does need to have the `-u` flag.

Reply:
@mgienow, the `-u` is short for `--set-upstream`. This snippet is correct!